lib/web.ts

```diff
@@ -583,6 +590,7 @@ export default class AutoConsent {
                 // if the cosmetic filters are actually working, report the hidden popup to the background.
                 // This may still be overridden later if an autoconsent rule matches.
                 // this may be a false positive: sometimes filters hide unrelated elements that are not cookie pop-ups
+                // @ts-expect-error - styles is defined at this point
```

Comment:
It won't be defined if `BUNDLE_FILTERLIST` is falsey.

Reply:
I added some more checks for BUNDLE_FILTERLIST that fixed the warning. This is a compile-time macro, so it should all be removed in minified builds